.github/workflows/deploy_branch_maven.reusable.workflow.yml

```diff
@@ -144,21 +176,38 @@ jobs:
           git_tag: ${{ inputs.git_tag }}
           git_tag_fail_if_exists: 'false'
           git_delete_tag_if_exists: 'true'
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+          github_token: ${{ secrets.userToken }}
 
       # Docker
-      - name: Docker build tag push
+      - name: Docker build
+        id: docker_build_tag_push
         # Run docker build only when a docker file has been specified (not empty value)
         if: ${{ !cancelled() && !failure() && inputs.docker-file != '' }}
         uses: ./.github/symphony-devops-workflows/actions/docker_build_tag_push
         with:
           docker-file: ${{ inputs.docker-file }}
           docker-tag-push: ${{ inputs.docker-tag-push }}
-          docker-tags: 'snapshot,${{ steps.get_project_version.outputs.project-version }}.${{ steps.compute_build_number.outputs.build-number }}'
-          docker-registry-name: 'registry.vnocsymphony.com'
-          docker-registry-project-name: ${{ inputs.docker-registry-project-name }}
-          docker-registry-microservice-name: ${{ inputs.symphony-microservice-name }}
+          docker-tags: '${{ steps.compute_docker_tags.outputs.DOCKER_TAGS_ALL }}'
+          docker-image-prefix: '${{ steps.compute_docker_tags.outputs.DOCKER_IMAGE_TAG_PREFIX }}'
 
+      - name: Kubernetes update deployment
+        if: ${{ !cancelled() && !failure() && inputs.kubernetes-update-deployment-descriptor && inputs.docker-file != '' && steps.docker_build_tag_push.outputs.docker-pushed == 'true' }}
+        uses: ./.github/symphony-devops-workflows/actions/kubernetes_update_deployment
+        with:
+          # Path is relative to deployer project root
+          deployment-file: '${{ inputs.symphony-microservice-name }}/deploy-service.yaml'
+          # New value for docker image in deployment file
+          docker-image: '${{ steps.compute_docker_tags.outputs.DOCKER_IMAGE_TAG_PREFIX }}:${{ steps.compute_docker_tags.outputs.DOCKER_TAG_WITH_VERSION }}'
+          fail-if-no-file-change: 'true'
+          # WARNING : relative path to current workspace, without leading "./" (for some reason this breaks replacement)
+          workspace-checkout-dir: 'symphony-kubernetes-deployer'
+          # The revision (branch) to be updated
+          symphony-kubernetes-deployer-ref: 'dev'
```

Comment:
is this correct? shouldn't be `develop`?

Reply:
Yes its on purpose, to avoid cluttering the develop branch for now, it represents the "dev" environment.
Might be changed in the future